building-with-containers/README.md

```diff
@@ -0,0 +1,245 @@
+---
+author: Laura
+language: go
+provider: azure-native
+use: first-party
+last-ran: 2 Nov 2022
+boilerplate: none
+last-host: Laura
+learn-content: none
+original-repo: none
+status: active
+---
+# Building with Containers on Azure
+
+Let’s explore more about building up a containerized microservices architecture on Azure using infrastructure as code and cloud engineering principles. We’ll use Go to build up our new architecture, and we’ll explore more about stacks, inputs and outputs, secrets, and more.
+
+### Prerequisites
+
+You will need the following tools:
+
+* [Pulumi CLI](https://www.pulumi.com/docs/get-started/install/?utm_source=da&utm_medium=referral&utm_campaign=workshops&utm_content=ced-fall2022-workshops)
+* [Pulumi FREE SaaS Account](https://app.pulumi.com/signup/?utm_source=da&utm_medium=referral&utm_campaign=workshops&utm_content=ced-fall2022-workshops)
+  * Pulumi state storage is FREE for developers, you can use this account for all your personal Pulumi projects and you never need to worry about where to store your state 😃
+* [Go 1.19.x](https://www.pulumi.com/docs/intro/languages/go/?utm_source=da&utm_medium=referral&utm_campaign=workshops&utm_content=ced-fall2022-workshops)
+* An [Azure account](https://www.pulumi.com/registry/packages/azure-native/installation-configuration/#credentials?utm_source=da&utm_medium=referral&utm_campaign=workshops&utm_content=ced-fall2022-workshops). The free tier should be all we need, but any costs accrued will be minimal if any appear at all.
+* [Docker](https://docs.docker.com/get-docker/) set up on your machine to build the container.
+
+### About this workshop
+
+In this workshop, we will explore secrets, stacks, inputs and outputs, and more.
+
+## Lab 1 - Setup
+
+Let's set up our sandbox application using a [Pulumi template for containers on Azure with Go](). First, make a new directory, and change into it.
```

Comment:
There may be a missing link here.

Reply:
Ha, caught. Yep; fixing now. Thanks!

Reply:
Note: The updated link in the incoming commit will be correct once https://github.com/pulumi/docs/pull/8182 actually lands.